bindings/node/src/index.ts

```diff
@@ -56,21 +60,30 @@ export interface MongoCrypt {
       expressionMode: boolean;
     }
   ): MongoCryptContext;
-  makeDecryptionContext(buffer: Buffer): MongoCryptContext;
-  makeExplicitDecryptionContext(buffer: Buffer): MongoCryptContext;
+  makeDecryptionContext(buffer: Uint8Array): MongoCryptContext;
+  makeExplicitDecryptionContext(buffer: Uint8Array): MongoCryptContext;
   makeDataKeyContext(
-    optionsBuffer: Buffer,
+    optionsBuffer: Uint8Array,
     options: {
-      keyAltNames: Buffer[];
-      keyMaterial: Buffer;
+      keyAltNames?: Uint8Array[];
+      keyMaterial?: Uint8Array;
     }
-  ): void;
-  makeRewrapManyDataKeyContext(filter: Buffer, encryptionKey: Buffer): void;
+  ): MongoCryptContext;
+  makeRewrapManyDataKeyContext(filter: Uint8Array, encryptionKey?: Uint8Array): MongoCryptContext;
   readonly status: MongoCryptStatus;
   readonly cryptSharedLibVersionInfo: {
     version: bigint;
     versionStr: string;
   } | null;
 }
 
+export type ExplicitEncryptionContextOptions = NonNullable<
```

Comment:
Can we just flip the way this is done, move the options in here, and use the type in the function? (same for `DataKeyContextOptions`?)

Reply:
Loosely held opinion. I think it is easier for the uninitiated to read types declared under the named type rather than from a calculation of its location in another type.